analysis/tvla.py

```diff
@@ -457,6 +457,9 @@ def run_tvla(ctx: typer.Context):
             else:
                 sample_step_hist = 5
 
+        # Number of traces after filtering over all steps.
+        num_traces_used_total = 0
```

Comment:
I'm not sure that this value should always be zero.
The condition of this if branch is:
```
if (cfg["input_histogram_file"] is None or cfg["output_histogram_file"] is not None) \
            and cfg["ttest_step_file"] is None:
```
so it is possible that `cfg["input_histogram_file"] is not None` (e.g. when we specify both input and output histogram files),
so the initial value for `num_traces_used_total` should be the number of traces used in the input histogram file.

Reply:
Good point, thanks! However, do we actually read the number of traces of the histogram when `cfg["input_histogram_file"]` is provided? Should this be done here:
https://github.com/lowRISC/ot-sca/blob/6bbb5aca447b65a6a48f885a87f2f377a570ff88/analysis/tvla.py#L328

Reply:
I think you're both somewhat right here. The histograms are defined as follows:
```
                # For every time sample we make two histograms, one for Hamming weight of the
                # sensitive variable = 0 (fixed set) and one for Hamming weight > 0 (random set).
                # histograms has dimensions [num_rnds, num_bytes, 2, num_samples, trace_resolution]
                # The value stored in histograms[v][w][x][y][z] shows how many traces have value z
                # at sample y, given that HW(state byte w in AES round v) = 0 (fixed set, x = 0) or
                # > 0 (random set, x = 1).
```
When an input histograms file is used, the number of traces used can be computed as sum(histograms[:][:][:][:][0][:]).

Since we already didn't set `num_traces_used_total` correctly when loading histograms before this PR. I suggest to create a new issue for tracking and merge this on as is.

Reply:
I agree that this can be merged as is.
I've added an item to issue #72 to track this.